src/api/getWilayaByZipCode.js

```diff
@@ -0,0 +1,16 @@
+const getWilayaByZipCode = data =>
+/**
+   * Takes a zip code (postal code) and returns it's wilaya.
+   *
+   * @example <p> Get 1000 wilaya
+   *
+   * //returns {mattricule: '1', name: 'Adrar', ...} because the zip code 1000 belongs to Adrar
+   * getWilayaByZipCode(31)
+   *
+   * @param { Number } zipCode postal code
+   * @returns { Object | undefined } Returns the target object, or undefined
+   */
+
+  (zipCode) => data.find(w => w.postalCodes.includes(zipCode));
```

Comment:
I'd prefer that we pass the result to [projectWilaya](https://github.com/dzcode-io/leblad/blob/1ca74c7ecb42ea58516e4bc0235c4abfebb5e6cb/src/utils/projections/wilayaProjection.js#L12).

As it was done here for example: 
https://github.com/dzcode-io/leblad/blob/17eccd42ded0eaaf1cc592a215ea1627d91a2551/src/api/getWilayaByCode.js#L14

Reply:
@Fcmam5  Done. I am assuming that from now on, each API should use the projection feature, right?